src/tests.rs

```diff
@@ -810,7 +856,7 @@ async fn exports_imported_earlier_default() {
       "location": {
         "filename": "file:///foo.ts",
         "line": 1,
-        "col": 0
+        "col": 6
```

Comment:
The variable declaration starts are now actually at the variable declaration starts instead of the variable statement.